container/kvm/cache_internal_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package kvm
+
+import (
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/juju/juju/environs/imagedownloads"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+)
+
+// internalCacheSuite is gocheck boilerplate.
+type internalCacheSuite struct{}
+
+var _ = gc.Suite(internalCacheSuite{})
```

Comment:
I think you need & here: var _ = gc.Suite(&internalCacheSuite{})

How does it run without? :)